packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -56,6 +56,15 @@ const PersonalDetailsForm = props => {
 
     const { errors, touched, values, setFieldValue, handleChange, handleBlur, setFieldTouched } = useFormikContext();
 
+    const handleToolTipStatus = React.useCallback(() => {
```

Comment:
```suggestion
    const handleToolTipStatus = useCallback(() => {
```
 should we change it here?

Reply:
The removal will be done in  different card. To reduce the size of changes, we have split those focusing on each section individually per card. In this PR u see some changes regd removal of react just to check if the config changes work :)